pages/common/montage.md

```diff
@@ -0,0 +1,24 @@
+# montage
+
+> Imagemagick image montage tool.
+> Tiles images into a customisable grid.
+
+- Tile images into a grid, automatically resizing images larger than the grid size:
+
+`montage {{image1.png}} {{image2.jpg}} {{imageN.png}} montage.jpg`
+
+- Tile images into a grid, automatically calculating the grid cell size from the largest image:
+
+`montage {{image1.png}} {{image2.jpg}} {{imageN.png}} -geometry +0+0 montage.jpg`
+
+- Set the grid cell size and resize images to fit inside their grid cells before tiling:
+
+`montage {{image1.png}} {{image2.jpg}} {{imageN.png}} -geometry 640x480+0+0 montage.jpg`
+
+- Limit the number of row and columns in the grid, causing input images to overflow into multiple output montages:
+
+`montage {{image1.png}} {{image2.jpg}} {{imageN.png}} -geometry +0+0 -tile 2x3 montage_%d.jpg`
+
+- Resize and crop images to completely fill their grid cells before tiling:
```

Comment:
I don't think this could be much clearer :) We could try to explain what `-gravity` means here, but the description would get too verbose.